automerge-cli/src/export.rs

```diff
@@ -0,0 +1,42 @@
+use crate::error::AutomergeCliError;
+use automerge_backend::AutomergeError;
+use automerge_protocol::Patch;
+use std::fs::File;
+use std::path::Path;
+use std::{io, io::Read};
+use serde_json::Value;
+
+fn get_changes(changes_file: &Path) -> io::Result<Vec<u8>> {
```

Comment:
I think this can be replaced by https://doc.rust-lang.org/std/fs/fn.read.html

Reply:
⚡ f176d45

Reply:
:boom: